### PR TITLE
translations ru,uk and fixed messages for translation

### PR DESCRIPTION
**Pull request by sergey-panasenko**
Hi Batiste
I am very inspired by your CMS, everything is done the way I wanted, or much better. I made a translation into Ukrainian, supplemented the translation into Russian, corrected the translation of the name of the application, models and fields, added the ability to translate fields and sections in placeholders.

### CODE REVIEW

File: pages/placeholders.py [hunk 1]
```diff
@@ -166,8 +166,10 @@ def get_field(self, page, language, initial=None):
         else:
             help_text = ''
         widget = self.get_widget(page, language)
+        print(self.ctype)
```
Comment (batiste) [inline pages/placeholders.py]: Forgot a print here

File: pages/models.py [hunk 2]
```diff
@@ -99,8 +99,8 @@ class Page(MPTTModel):
             blank=True)
 
     freeze_date = models.DateTimeField(_('freeze date'),
-            null=True, blank=True, help_text=_('''Don't publish any content
-            after this date.'''))
+            null=True, blank=True, help_text=_("Don't publish any content "
+            "after this date."))
```
Comment (batiste) [inline pages/models.py]: Is there a reason for changing the string here? Is it a best practice?

File: pages/placeholders.py (hunk 1 above)
Comment (sergey-panasenko) [reply pages/placeholders.py]: Fixed

File: pages/models.py (hunk 2 above)
Comment (sergey-panasenko) [reply pages/models.py]: I just wanted to escape newline and tabs in string, so it is not showing in localization editor.

**Pull request merged**